api/client.yaml

```diff
@@ -996,6 +994,14 @@ components:
         match:
           type: number
           example: 0.91
+    OfacType:
```

Comment:
This should probably be called `SdnType`

Reply:
I originally had it as `SdnType`, but noticed `SSI` uses the same type
https://github.com/moov-io/watchman/blob/a0df66ec303b421fe5e488e00c165320c3182a08/client/api/openapi.yaml#L1442

Reply:
I don't believe the SSI list contains vessels right now. It's a different list, so I'd have `SdnType` and `SsiType`. 

https://home.treasury.gov/policy-issues/financial-sanctions/consolidated-sanctions-list/sectoral-sanctions-identifications-ssi-list

Reply:
good point, I'll do that